sites/next.skeleton.dev/src/examples/resources/cookbook/image-layouts/ExampleQuad.astro

```diff
@@ -0,0 +1,18 @@
+<!-- Flex -->
+<section class="flex justify-center">
```

Comment:
Similar to Featured, let's move the grid styles up to the wrapping `section` and remove this extra dive. It's redundant.